.github/workflows/web_client_merge.yml

```diff
@@ -0,0 +1,50 @@
+name: Web client CD
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'web/client/**'
+
+jobs:
+
+  web_cd:
+    name: Deploy web client
+    runs-on: ubuntu-latest
+
+    env:
+      ENV_NAME              : prd
+      CLOUDSDK_CORE_PROJECT : web-based-gtfs-validator
+
+    steps:
+
+      - uses: actions/checkout@v3
+        with:
+          # We need to download all tags so that the axion-release-plugin
+          # can resolve the most recent version tag.
+          fetch-depth: 0
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          # We need a recent version of Java with jpackage included.
+          java-version: '17'
+          # We use the zulu distribution, which is an OpenJDK distro.
+          distribution: 'zulu'
+
+      # for npm
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - uses: google-github-actions/setup-gcloud@v0
```

Comment:
the v1 version drops support for configuring service account credentials and the gcloud command suite in a single action, and requires the addition of the `google-github-actions/auth` action to configure the service account prior to running the `google-github-actions/setup-gcloud` step. Decoupling these steps doesn't benefit our use case at all, and it remains simpler to continue using the `v0` action, which has no tangible downside.

Reply:
Hi @lottspot I understand the challenge here on implementation. However, starting a new development with an already deprecated feature is a red flag for me. My understanding is that v1 is based on `Workload Identity Pools`, removing the need for long-lived access tokens. This is a gain in security. In addition, we will need to upgrade this in the future to keep our CI up to date with performance and security fixes. I would prefer to address it now instead of creating a tech debt from the start.

Reply:
It's worth noting that implementing the `v1` action does not in it of itself grant this security gain; configuring workload identity pools required discrete provider-side configuration which we have not performed for this project. It will still be using a long-lived `.json` key credential. I'll go ahead and update the action to use `v1` though in alignment with your preferences.

Reply:
I appreciate your understanding. Let's go with the `v1`; we can always set up the `Workload Identity Pool` later to harden the security of Github and GCP integration.